docs/tutorial/population_dynamics.rst

```diff
@@ -151,7 +151,7 @@ physical state of the system is hidden or unobservable.
 This description is the most basic implementation of an HMM. For detailed theoretical and practical
 explanations on the use and training of HMMs see the seminal work by Rabiner :cite:`Rabiner1989HMM`.
 
-We can fit an HMM with a single line of code::
+Run the Baum-Welch algorithm to obtain the parameters of the Hidden Markov Model, such as determine the transition matrix:
```

Comment:
There seems to be a typo here? Also, I wonder whether it can be a little confusing, since the line of code presented here doesn't actually return the parameters itself, but a model.

I do agree that it's nice to mention the algorithm, in case people are interested. Maybe something like?

```suggestion
Fit an HMM using the Baum-Welch algorithm::
```

And then mention later that the estimated parameters can be extracted (for instance, the transition matrix is available through `hmm.transition_matrix`).

```rst
This algorithm estimates the model parameters based on the provided data.
One of these estimated parameters is the transition matrix which can be extracted as::

    >>> hmm.transition_matrix
```

Reply:
Not sure if this was intentional, but did you decide that the second part (mentioning the transition matrix being estimated and how to obtain it) is not so useful?

Reply:
I actually think mentioning the transition matrix is very useful. I thought I just committed your suggestions, but may have missed this one. I did not have time yet to carefully look at the PR and finalize it.

Reply:
Ahh okay, I wasn't sure whether it was ready for rereview yet. Since the comments were resolved I thought you might have forgotten to rerequest review. But yeah, take your time 👍